lib/plugins/grafana/index.js

```diff
@@ -1,11 +1,57 @@
 'use strict';
+const path = require('path');
 const sendAnnotations = require('./send-annotation');
 const tsdbUtil = require('../../support/tsdbUtil');
 const throwIfMissing = require('../../support/util').throwIfMissing;
+
 const defaultConfig = {
   port: 80
 };
+
 module.exports = {
+  name() {
+    return path.basename(__dirname);
+  },
+  options() {
```

Comment:
What about move those options out to a new file (cli.js or something) and set a new standard for that also? For some plugins the index.js is cluttered I'm thinking about minimising the code in there?

Reply:
I updated with the following:

* Renamed the proposed method to `cliOptions()` to be more explicit and to not shadow/override the property defined in the `.open()` method.
* Moved all definitions to `grafana/cli.js` that's later on `require`d by the method/property above

Not all plugins will have extensive options (like `browsertime`) and I think it's very convenient to keep the plugin interface clear with an explicit method/property like `cliOptions()`